From 0a2db6b2cae01b7951d66eade616161533ba1133 Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Sat, 23 Feb 2013 16:24:52 +0200
Subject: [PATCH 8/8] man: escape quote properly in header

Without this patch

LC_ALL=en_US.UTF-8 MANROFFSEQ='' MANWIDTH=80 man --warnings -E UTF-8 -l -Tutf8 -Z /usr/share/man/man3/tapset::ansi.3stap.gz >/dev/null

prints

<standard input>:1: warning: macro `"' not defined
---
 doc/SystemTap_Tapset_Reference/manpager |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/doc/SystemTap_Tapset_Reference/manpager b/doc/SystemTap_Tapset_Reference/manpager
index 30e9571..a250162 100755
--- a/doc/SystemTap_Tapset_Reference/manpager
+++ b/doc/SystemTap_Tapset_Reference/manpager
@@ -60,7 +60,7 @@ done ;
 
 # create man page headers
 for i in `ls | grep -v .stp | grep -v tapsetdescription` ; do
-  echo ".\" -*- nroff -*-" >> $i.template ;
+  echo ".\\\" -*- nroff -*-" >> $i.template ;
   echo ".TH tapset::"$i" 3stap --- IBM" >> $i.template ;
   echo ".SH NAME" >> $i.template ;
   echo "tapset::"`cat $i.stp`" \- systemtap "`cat $i.stp`" tapset" >> $i.template ;
-- 
1.7.2.5

